### PR TITLE
[enriched-jira] Add comments to enriched index

### PR DESCRIPTION
**Pull request by valeriocos**
This code includes Jira comments to the enriched index. To do this, changes have been made to allow the cohesistance of both comments and issues in the same index, following an approach similar to meetup and stackexchange enrichers. Thus, enriched issues and commits have different ids, the former is the issue
ids, while the latter is a str concatenating the issue id and the comment id.

Signed-off-by: Valerio Cosentino <valcos@bitergia.com>

### CODE REVIEW

**Review by sduenas — APPROVED** [review]

File: grimoire_elk/enriched/jira.py [hunk 1]
```diff
@@ -282,11 +340,102 @@ def get_rich_item(self, item):
...
+        for comment in comments:
+            ecomment = {}
+
+            for f in self.RAW_FIELDS_COPY:
+                ecomment[f] = eitem[f]
+
+            # Copy data from the enriched issue
+            ecomment['project_id'] = eitem['project_id']
+            ecomment['project_key'] = eitem['project_id']
+            ecomment['project_name'] = eitem['project_id']
```
Comment (sduenas) [inline grimoire_elk/enriched/jira.py]: All these values should have the same value?

**Review by sduenas — COMMENTED** [review]
I'm not sure if `project_id`, `project_key` and `project_name` should have the same value. In this case the value of `project_id` field. Is this the case?

I think the issue has different values for these fields.

**Comment by valeriocos** [comment]
my fault, copy/paste and i didn't replace the values. thank you for pointing this out, i'll modify it asap

**Comment by valeriocos** [comment]
@sduenas please re-check, thanks!

**Review by sduenas — APPROVED** [review]
LGTM

**Pull request merged**